editor/src/components/canvas/canvas-strategies/post-action-options/post-action-options.ts

```diff
@@ -79,3 +92,34 @@ export function createPasteToReplacePostActionActions(
   }
   return null
 }
+
+export function createNavigatorReparentPostActionActions(
+  dragSources: Array<ElementPath>,
+  targetParent: ElementPath,
+  indexPosition: IndexPosition,
+  canvasViewportCenter: CanvasPoint,
+  jsxMetadata: ElementInstanceMetadataMap,
+  allElementProps: AllElementProps,
+): Array<EditorAction> {
+  const navigatorReparentPostActionMenuData: NavigatorReparentPostActionMenuData = {
+    type: 'NAVIGATOR_REPARENT',
+    dragSources: dragSources,
+    targetParent: targetParent,
+    indexPosition: indexPosition,
+    canvasViewportCenter: canvasViewportCenter,
+    jsxMetadata: jsxMetadata,
+    allElementProps: allElementProps,
+  }
+
+  const defaultChoice =
+    NavigatorReparentPropsReplacedPostActionChoice(navigatorReparentPostActionMenuData) ??
```

Comment:
as I wrote in slack, I think 
ReplacePropsNavigatorReparentPostActionChoice and PreservePropsNavigatorReparentPostActionChoice would read much better because the first word is what distinguishes the two

Reply:
That makes sense, I renamed all of our paste and navigator reparent post action choices in 22a35e45d. It looks better almost all places, the name is still weirdly too long :)